pending-documentation.md

```diff
@@ -185,6 +185,99 @@ have any constructors declared.
 syntax Bytes [hook(BYTES.Bytes), token]
 ```
 
+### Converting between `[token]` sorts
+
+You can convert between tokens of one sort via `String`s by defining functions
+implemented by builtin hooks.
+The hook `STRING.token2string` allows conversion of any token to a string:
+
+```k
+syntax String ::= FooToString(Foo)  [function, functional, hook(STRING.token2string)]
+```
```

Comment:
Should this be indented consistently with the other code blocks?

Reply:
Different code blocks in the document are formatted differently. The most common was unindented, so I was using that. But, I'll change it to 4 spaces.

Reply:
Actually, the document is overwhelmingly unindented, so I'll move the couple I forgot to left indented to that format